README.md

```diff
@@ -49,21 +50,30 @@ AWS is our main focus at the moment, but we plan to support GCP, Azure and other
 
 ### AWS services with significant API coverage
 
+- CloudWatch
+- CodeBuild
+- CodeDeploy
+- CodePipeline
```

Comment:
We have temporarily disabled this modules until @aguillenv 's Two-Way mode work is done, so I'd remove this.

```suggestion
```